assets/js/modules/search-console/components/dashboard/SearchFunnelWidget/Overview.js

```diff
@@ -238,24 +248,44 @@ const Overview = ( {
 							</Cell>
 
 							<Cell { ...quarterCellProps }>
-								{ viewContext === VIEW_CONTEXT_DASHBOARD && (
-									<DataBlock
-										stat={ 3 }
-										className="googlesitekit-data-block--goals googlesitekit-data-block--button-4"
-										title={ __(
-											'Goals',
-											'google-site-kit'
-										) }
-										datapoint={ analyticsGoalsDatapoint }
-										change={ analyticsGoalsChange }
-										changeDataUnit="%"
-										context="button"
-										selected={ selectedStats === 3 }
-										handleStatSelection={
-											handleStatsSelection
-										}
-									/>
-								) }
+								{ viewContext === VIEW_CONTEXT_DASHBOARD &&
+									( ! analyticsGoalsData?.items?.length ? (
+										<CTA
+											title={ __(
+												'Use goals to measure success',
+												'google-site-kit'
+											) }
+											description={ __(
+												'Goals measure how well your site or app fulfills your target objectives',
+												'google-site-kit'
+											) }
+											ctaLink={ supportURL }
+											ctaLabel={ __(
+												'Create a new goal',
+												'google-site-kit'
+											) }
+											ctaLinkExternal
+										/>
+									) : (
+										<DataBlock
+											stat={ 3 }
+											className="googlesitekit-data-block--goals googlesitekit-data-block--button-4"
+											title={ __(
+												'Goals',
+												'google-site-kit'
+											) }
+											datapoint={
+												analyticsGoalsDatapoint
+											}
+											change={ analyticsGoalsChange }
+											changeDataUnit="%"
+											context="button"
+											selected={ selectedStats === 3 }
+											handleStatSelection={
+												handleStatsSelection
+											}
+										/>
+									) ) }
```

Comment:
Ternary statements, especially ones this long, can be hard to read/decipher/quickly scan in JSX. We're in the habit of not using ternary statements and instead using two different statements wherever possible.

For this section, can we do something like:

```js
! analyticsGoalsData?.items?.length && <CTA />
analyticsGoalsData?.items?.length > 0 && <DataBlock />
```

That's a bit more explicit, but it's also easier to scan quickly. 🙂

Reply:
Sounds good 👍